### ISSUE TITLE
add functionality to auto click on the center of the page

### ISSUE THREAD

**Issue by nerdynikhil** [nerdynikhil/couch-voyager#1]

**Comment by Harsh3363** [comment]
Hi @nerdynikhil assign this to me I would like to add this functionality.

**Comment by nerdynikhil** [comment]
@Harsh3363 Assigned. Please go ahead.

**Comment by Dydipanshu** [comment]
Hi @nerdynikhil can you please tell the time interval of auto click.

**Comment by nerdynikhil** [comment]
let’s go with 5 seconds for now?

**Comment by Harsh3363** [comment]
Hi @nerdynikhil I'm not able to use the extension I have changed manifest_version to 3 as 2 was deprecated but after loading the extension still I'm not able to use it.

**Comment by nerdynikhil** [comment]
could you please specify the issue you are facing? it loads?

**Comment by Harsh3363** [comment]
Yeah i was able to load it but not able to access it on the website
![issue](https://github.com/nerdynikhil/couch-voyager/assets/72756692/92b7dfd9-0c4e-4b94-8de2-61e6803dfa79)

**Comment by nerdynikhil** [comment]
that looks new to me. let's setup some time to go over this in a meet. please ping me on discord `nerdynikhil` with your availability

**Comment by Harsh3363** [comment]
> that looks new to me. let's setup some time to go over this in a meet. please ping me on discord `nerdynikhil` with your availability

Done

**Comment by rajanarahul93** [comment]
Is this issue still open ?

**Comment by nerdynikhil** [comment]
Yes, it is.
